[2023-05-24T18:45:58.213+0200] {processor.py:156} INFO - Started process (PID=7930) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:45:58.215+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T18:45:58.216+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:45:58.216+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:45:58.220+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:45:58.220+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T18:45:58.221+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:45:58.221+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T18:45:58.221+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:45:58.239+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2023-05-24T18:46:51.828+0200] {processor.py:156} INFO - Started process (PID=7975) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:46:51.867+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T18:46:51.868+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:46:51.868+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:46:51.873+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:46:51.872+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T18:46:51.874+0200] {logging_mixin.py:149} INFO - [2023-05-24T18:46:51.873+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T18:46:51.874+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T18:46:51.898+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.074 seconds
[2023-05-24T21:47:30.918+0200] {processor.py:156} INFO - Started process (PID=8092) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:47:30.920+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T21:47:30.921+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:47:30.920+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:47:30.924+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:47:30.924+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T21:47:30.925+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:47:30.924+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T21:47:30.925+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:47:30.938+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2023-05-24T21:48:19.207+0200] {processor.py:156} INFO - Started process (PID=8142) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:48:19.243+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T21:48:19.245+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:48:19.244+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:48:19.250+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:48:19.249+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T21:48:19.250+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:48:19.250+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T21:48:19.250+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:48:19.271+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.069 seconds
[2023-05-24T21:49:18.630+0200] {processor.py:156} INFO - Started process (PID=8198) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:49:18.631+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T21:49:18.632+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:49:18.632+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:49:18.635+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:49:18.634+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T21:49:18.635+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:49:18.635+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T21:49:18.635+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:49:18.647+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.019 seconds
[2023-05-24T21:50:07.403+0200] {processor.py:156} INFO - Started process (PID=8246) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:07.404+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T21:50:07.404+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:07.404+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:07.406+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:07.406+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T21:50:07.406+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:07.406+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T21:50:07.407+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:07.415+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.014 seconds
[2023-05-24T21:50:53.312+0200] {processor.py:156} INFO - Started process (PID=8293) to work on /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:53.314+0200] {processor.py:799} INFO - Processing file /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2023-05-24T21:50:53.315+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:53.315+0200] {dagbag.py:541} INFO - Filling up the DagBag from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:53.318+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:53.317+0200] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2023-05-24T21:50:53.318+0200] {logging_mixin.py:149} INFO - [2023-05-24T21:50:53.318+0200] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2023-05-24T21:50:53.319+0200] {processor.py:811} WARNING - No viable dags retrieved from /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2023-05-24T21:50:53.328+0200] {processor.py:178} INFO - Processing /home/choco/.local/lib/python3.10/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
